src/Libraries/RevitNodes/GeometryConversion/UnitConverter.cs

```diff
@@ -1,28 +1,31 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+
 using Autodesk.DesignScript.Runtime;
 using Autodesk.Revit.DB;
-using DynamoUnits;
+
+using RevitServices.Persistence;
 
 namespace Revit.GeometryConversion
 {
     [SupressImportIntoVM]
     public static class UnitConverter
     {
-        public static double DynamoToHostFactor
+        public static double DynamoToHostFactor(UnitType unitType)
         {
-            get { return Length.FromDouble(1.0).ConvertToHostUnits(); }
+            var revitDisplayUnits =
+                DocumentManager.Instance.CurrentDBDocument.GetUnits()
+                    .GetFormatOptions(unitType)
+                    .DisplayUnits;
+            return UnitUtils.ConvertToInternalUnits(1, revitDisplayUnits);
```

Comment:
That "magic" number is actually the number I want. In order to use this API, we need to provide a value to be converted. That value, in this case, is 1.0.